.NET/Microsoft.Recognizers.Text.DateTime/Utilities/TimexUtility.cs

```diff
@@ -88,5 +93,50 @@ public static string GenerateYearTimex(DateObject date = default(DateObject))
                 return date.Year.ToString("D4");
             }
         }
+        
+        public static DatePeriodTimexType GetDatePeriodTimexType(string durationTimex)
+        {
+            var minimumUnit = durationTimex.Substring(durationTimex.Length - 1);
+            var ret = DatePeriodTimexType.ByDay;
+
+            if (minimumUnit == "Y")
```

Comment:
can we put "Y" to somewhere to be some constants

Reply:
Fixed.